docs/packages.md

```diff
@@ -315,11 +315,12 @@ You can set default parameters for all the entities in a package. You do this by
 2. Display the parameters in the package and action, and see how the `identity` action in the package inherits parameters from the package.
 
   ```
-  $ wsk package get custom parameters
+  $ wsk package get custom
```

Comment:
i saw you opened an issue to fix the property projection - should we link the issue to these changes so we can revert them later?